ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -327,6 +354,15 @@ export default class SignatureRequestOriginal extends Component {
           </div>
         ) : null}
         {this.renderFooter()}
+        {messagesCount > 1 ? (
+          <Button
+            type="link"
+            className="request-signature__container__reject"
+            onClick={() => this.handleCancelAll()}
+          >
+            {rejectNText}
+          </Button>
+        ) : null}
```

Comment:
```suggestion
        ) :<></>}
```

Reply:
@mrbaguvix updated.

Reply:
@brad-decker done.

Reply:
Oh my bad, just noticed you're using the good 'ol _class-based_ component. `null` is good.